scripts/contractInteraction/governance.py

```diff
@@ -20,4 +20,11 @@ def queueProposal(id):
 def executeProposal(id):
     governor = Contract.from_abi("GovernorAlpha", address=conf.contracts['GovernorOwner'], abi=GovernorAlpha.abi, owner=conf.acct)
     tx = governor.execute(id)
-    tx.info()
\ No newline at end of file
+    tx.info()
+
+def transferLockedSOVOwnershipToGovernance():
```

Comment:
to transfer ownership we also need to remove other admins after adding the governor admin

Reply:
@cowsant could you explain here the privileges of the admin of the LockedSOV contract?

Reply:
Admin has rights such as:
- Add another admin
- Remove existing admin
- update the Vesting Registry, Duration and Cliff.
- start the process of migration to new contract.

Reply:
OK. could you verify if there are any other admins other than exchequer? exchequer needs to remain until after governor admin takes over at least.

Reply:
I verified no other admins other than exchequer right now.